src/IntegrationUtils.ts

```diff
@@ -265,17 +265,14 @@ async function handleDefinedTask(context: vscode.Uri) {
 
 function getSelectedConfigMap(): Promise<string | undefined> {
 	return new Promise <string | undefined>( async (resolve, reject) => {
-		await getConfigMaps()
-			.then( (configMaps) => {
-				vscode.window.showQuickPick(configMaps, {
+		const configMaps: string[] = await getConfigMaps();
+		if(configMaps.length === 0) {
+			reject(`No ConfigMap available.`);
+		}
+		return resolve(await vscode.window.showQuickPick(configMaps, {
 					placeHolder: 'Select an available Kubernetes ConfigMap or ESC to cancel'
-				}).then ( (selectedConfigMap) => {
-					resolve(selectedConfigMap);
-				});
-			}).catch ( (error) => {
-				reject(new Error(`No ConfigMaps available: ${error}`));
-			});
-		});
+				}));
+	});
```

Comment:
this modification simplifies a bit the code and seems to be required to be able to mock showQuickPick